app/utils/dictionary/export-event.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
This is just dummy data. Don't put it in dictionary. Just put it in the component itself for now as a string. Make the object smaller. Just put a name, starts_at and ends_at ... it's just a dummy.